pages/404/index.tsx

```diff
@@ -12,7 +12,7 @@ export default function Custom404() {
         <title>404 Not Found</title>
       </Head>
       <Layout activePage="404">
-        <Heading level="one">404 Not Found</Heading>
+        <Heading level="h1">404 Not Found</Heading>
```

Comment:
is there a difference between providing `h1` and `one` to the level attribute?

Reply:
This is Diego's update in PR #37  but I wasn't pointing my PR to his. In the DS, the `h1` and `one` prop values are the same but with the Typography update, we decided to go with `h1` as the value name because it's clearer when writing HTML that it's an `h1`.